runtime/tutor.txt

```diff
@@ -415,6 +422,303 @@ _________________________________________________________________
+ 3. press d to delete your selection, breaking the sentence free
+    of the square brackets
+ 4. go to the very end of the line and repeat again with uppercase
+    F. Remember this time you're selecting to a closing ].
+
+ --> -----[This sentence is surrounded in square brackets, free me!]-----
+
+ Note: Unlike vim, helix doesn't limit this command to just one
+       line. It'll just search for that character in the file.
+
+
+
+
+
+=================================================================
+=                  SELECTING TIL' A CHARACTER                   =
+=================================================================
+
+ Select up to, but NOT including ('til) a character with t. Select backwards
+ until a character with uppercase T
+
+ 1. Move the cursor to the line below marked -->.
+ 2. press t and then A to select until A.
+ 3. press d to delete the dashes.
+ 4. Go to the line end, and repeat with T. Only
+    this time you're searching for an 's'
+
+ --> -----A sentence surrounded in dashes-----
+
+ Note: Unlike vim, helix doesn't limit this command to just one
+       line. It'll just search for that character in the file.
+
+
+
+
+
+=================================================================
+=                             RECAP                             =
+=================================================================
+
+ * Press C to copy the selection on the line below and Alt-C for
+   above.
+
+ * Press s to select all instances of a regex pattern inside
+   the current selection.
+
+ * Press semicolon ( ; ) to collapse
+
+ * Press f / F to extend selection up to & including a character.
+
+ * press t / T to extend selection until a character.
+
+
+
+
+
+=================================================================
+=                   COPYING AND PASTING TEXT                    =
+=================================================================
+
+ Press y to yank (copy) the selection to the clipboard.
+ Press p to paste the yanked selection after the cursor.
+ Press uppercase P to paste the yanked selection *before*
+ the cursor.
+
+ 1. Move the cursor to the line below marked -->.
+ 2. press w to select "banana" and y to yank it.
+ 3. move between "2" and "3" and press p to paste.
+ 4. Repeat between "3" and "4"
+
+
+ --> 1 banana 2 3 4
+
+ Note: Whenever you delete (d) or change (c) text, helix will
+       copy the deleted/changed text to the clipboard, you can
+       avoid this with alt-d and alt-c respectively.
+ Note: Helix doesn't share the system clipboard by default.
+       Press space-Y and space-P to yank/paste to/from
+       your computers main clipboard
+
+
+
+
+
+=================================================================
+=                        SWITCHING CASE                         =
+=================================================================
+
+ Press ~ to switch the case (capitalised-ness) of all selected
+ letters - or just the one under the cursor.
```

Comment:
Adding that it only changes the case of the letter under the cursor when there is no selection is redundant, as that is how every other command works as well.